static/app/components/button.spec.tsx

```diff
@@ -39,3 +27,17 @@ describe('Button', function () {
     expect(spy).not.toHaveBeenCalled();
   });
 });
+
+describe('LinkButton', function () {
+  it('renders react-router link', function () {
+    render(<LinkButton to="/some/route">Router Link</LinkButton>);
+  });
+
+  it('renders normal link', function () {
+    render(<LinkButton href="/some/relative/url">Normal Link</LinkButton>);
```

Comment:
this could render a text. shouldn't we verify if the  _button_ contains a _href_ and it is _active_?

Reply:
example:

`expect(screen.getByRole('button', {name: 'Normal Link'})).toHaveAttribute('href', '/some/relative/url');`

Reply:
loooks like it didn't before lol

Reply:
Well in this case it should be a `a` element, not a button 😉

Reply:
this tricked me - yes we should check for an `a` element 😄

Reply:
it is a double trick because we set `role="button"` on this lol